virttest/shared/cfg/base.cfg

```diff
@@ -1101,12 +1101,13 @@ uuid_dimm = ""
 # Note: The following types are supported: sev, snp, tdx
 #       Disable secure guest: vm_secure_guest_type[_vm1] = ""
 #
-# Set the secure guest object options, use a blank to join more options (optional)
-#vm_secure_guest_object_options[_vm1] = "debug=on mrconfigid=xx mrowner=yy"
+# Set the secure guest object options with json format (optional)
+#vm_secure_guest_object_options[_vm1] = json:{"attributes": 268435456, "quote-generation-socket": {"type": "unix", "path": "/run/tdx-qgs/qgs.socket"}}
```

Comment:
as we only support the json string setting for the vm_secure_guest_object_options param, we don't need to add the "json:" prefix